DataFormats/Common/interface/DetSet.h

```diff
@@ -66,6 +66,7 @@ namespace edm {
     const_reference operator[](size_type i) const { return data[ i ]; }
     void reserve(size_t s) { data.reserve(s); }
     void push_back(const T & t) { data.push_back(t); }
+    void emplace_back(const T & t) { data.emplace_back(t); }
```

Comment:
This is actually not the correct interface for `emplace_back`.
```suggestion
    template<class... Args>
    auto emplace_back(Args&&... args) { return data.emplace_back(std::forward<Args>(args)...); }
```

Reply:
Thanks a lot for your suggestion, @Dr15Jones ! Indeed, this was way too naive of me not to test it beforehand...